lib/src/test/java/uk/gov/di/ipv/core/library/service/IpvSessionServiceTest.java

```diff
@@ -152,8 +158,33 @@ void shouldCreateSessionItemForDebugJourney() {
                 ipvSessionItemArgumentCaptor.getValue().getUserState());
     }
 
+    @Test
+    void shouldCreateSessionItemForRefactorJourney() {
+        when(mockConfigService.getSsmParameter(ConfigurationVariable.JOURNEY_TYPE))
+                .thenReturn("ipv-core-refactor-journey");
+        IpvSessionItem ipvSessionItem =
+                ipvSessionService.generateIpvSession(
+                        SecureTokenHelper.generate(), null, false, null);
+
+        ArgumentCaptor<IpvSessionItem> ipvSessionItemArgumentCaptor =
+                ArgumentCaptor.forClass(IpvSessionItem.class);
+        verify(mockDataStore)
+                .create(ipvSessionItemArgumentCaptor.capture(), eq(BACKEND_SESSION_TTL));
+        assertNotNull(ipvSessionItemArgumentCaptor.getValue().getIpvSessionId());
+        assertNotNull(ipvSessionItemArgumentCaptor.getValue().getCreationDateTime());
+
+        assertEquals(
+                ipvSessionItemArgumentCaptor.getValue().getIpvSessionId(),
+                ipvSessionItem.getIpvSessionId());
+        assertEquals(
+                INITIAL_IPV_JOURNEY_STATE, ipvSessionItemArgumentCaptor.getValue().getUserState());
+        assertEquals(ipvSessionItem.getJourneyType(), IpvJourneyTypes.IPV_CORE_REFACTOR_JOURNEY);
```

Comment:
sonar wants you to swap these too args, so they match with `expected` and `actual`.